dali/base/dafdesc.cpp

```diff
@@ -1347,49 +1360,52 @@ class CFileDescriptor:  public CFileDescriptorBase, implements ISuperFileDescrip
                 fullpath.append(directory);
             replaceClusterDir(idx,copy, fullpath);
 
-            unsigned lcopy;
-            IClusterInfo * cluster = queryCluster(idx,copy,lcopy);
-            if (cluster)
+            if (pt.overridename.isEmpty())
```

Comment:
could add a comment to indicate this is adding a stripe directory, and this is not done if overridename is set